dask/array/tests/test_array_core.py

```diff
@@ -5045,6 +5045,7 @@ def test_from_array_meta():
 def test_compute_chunk_sizes():
     x = da.from_array(np.linspace(-1, 1, num=50), chunks=10)
     y = x[x < 0]
+    y._chunks = ((np.nan,) * 5,)
```

Comment:
Similar comments to below about how you should not include the slice in the line above. Also since we are explicitly setting the chunk size, it isn't really necessary to check it like we are doing in the two lines below.